Notify for kick and mute

damencho opened the pull request:
Depends on https://github.com/jitsi/lib-jitsi-meet/pull/915

damencho commented:
WIP 
Needs the mobile part of the muted notification.

virtuacoplenny commented on conference.js:
```diff
@@ -1855,6 +1855,12 @@ export default {
+            if (participantThatMutedUs) {
+                APP.UI.notifyMuted(participantThatMutedUs);
+            }
+        });
```
Could this be added somewhere else, maybe within the react files somewhere, so that more logic isn't added to conference.js? At the least I think we should avoid adding more to APP.UI.

damencho replied:
This was done while adding the mobile part.

damencho replied:
Not really, just the mute part, not the kick. I will update tomorrow and the kick part ...

virtuacoplenny commented on lang/main.json:
```diff
@@ -197,11 +197,11 @@
         "kickParticipantDialog": "Are you sure you want to kick this participant?",
         "kickParticipantTitle": "Kick this member?",
-        "kickTitle": "Kicked from meeting",
+        "kickTitle": "Ouch! __participantDisplayName__ kicked you out of the meeting",
```
"Ouch"? Where did this copy come from?

damencho replied:
We already had this in kickMessage.

damencho commented:
![image (1)](https://user-images.githubusercontent.com/3263098/59469385-96219800-8e2c-11e9-9c54-53d44180e9b9.png)
![Screenshot_20190613-222322](https://user-images.githubusercontent.com/3263098/59469392-9883f200-8e2c-11e9-8f84-67f5be260d44.png)

damencho commented:
![Screenshot_20190614-113840](https://user-images.githubusercontent.com/3263098/59504118-1af6cb00-8e9a-11e9-866e-0027ff3c1ea3.png)
![Screenshot_20190614-113805](https://user-images.githubusercontent.com/3263098/59504120-1d592500-8e9a-11e9-90e5-f06a81058d0a.png)

@virtuacoplenny Can you make another round. I moved everything to react and added the kick notifications for mobile.

damencho commented:
Seems tests are failing for both PRs very constantly :) so I will be looking at this next week

virtuacoplenny commented on react/features/base/participants/actions.js:
```diff
@@ -382,6 +386,51 @@ export function participantUpdated(participant = {}) {
+            titleKey: 'notify.mutedRemotelyTitle',
+            titleArguments: {
+                participantDisplayName:
+                    getParticipantDisplayName(getState, participant.getId())
```
Is the display name not already on the participant? It's better to get it from redux?

virtuacoplenny commented on react/features/base/participants/actions.js:
```diff
@@ -382,6 +386,51 @@ export function participantUpdated(participant = {}) {
+            return;
+        }
+
+        dispatch(showNotification({
```
Should this be a notification action then?
